autoapi/mappers/python/objects.py

```diff
@@ -183,10 +199,7 @@ def display(self):
 
         :type: bool
         """
-        if self.short_name == "__init__":
-            return False
-
-        return super(PythonMethod, self).display
+        return super(PythonMethod, self).display and self.short_name != "__init__"
```

Comment:
This might need to fire it's own version of the event if we want to be able to also pass the result of `self.short_name != "__init__"` into the `skip` argument of the event?

Reply:
I not really sure about this, generally i don't understand why this rule is hard-wired, previously i thought that it depends on `autoapi_python_class_content` option. Then i realized, how it works :)
In my opinion, class docstring and `__init__` docstring should be represented as different objects, what you think about?

Reply:
Usually `__init__` isn't displayed because the arguments of `__init__` are displayed in the signature of the class itself, and the arguments are documented in the class docstring. Sometimes this isn't desirable, which is why `autoapi_python_class_content` exists.
But now that that you're adding this new event, users will have the option to display `__init__` anyway if they want.
So I think it makes sense to hide `__init__` by default, but still ask your new event if it should be displayed.

Reply:
I got you, but still wandering: why you still want to merge class docstring with neither `__init__` or `__new__` methods docstrings? I think that depending on `autoapi_python_class_content` options they should be displayed independently. As example:

*class* **Foo**
&nbsp;&nbsp; Class docstring

&nbsp;&nbsp; **\_\_init\_\_(**_self, arg: bool_**)**
&nbsp;&nbsp;&nbsp;&nbsp; Additional doctring as well

Then it will be pretty obvious how to class methods :)

Reply:
Usually you don't use `__init__` directly (or `__new__`). You use them by the class itself. For example you use `instance = MyClass()` instead of `instance = MyClass.__init__()`. So it doesn't make sense to document it as a method.